lib/counter_web/live/counter_live.ex

```diff
@@ -0,0 +1,28 @@
+defmodule CounterWeb.CounterLive do
+  use Phoenix.LiveView
+
+  def mount(_session, _, socket) do
+    socket = assign(socket, :count, 0)
+    {:ok, socket}
+  end
+
+  def render(assigns) do
+    ~L"""
```

Comment:
@JuanMaGiraldo what this sigil means?

Reply:
It means that html code will be declared inside the quotes.